### DISCUSSION TITLE
More options for the DeepslateLayerDimensionBlacklist

### DISCUSSION THREAD

**Discussion by AetherLegacyPro** [Roadhog360/Et-Futurum-Requiem#277] [Ideas]
Add an option for the DeepslateLayerDimensionBlacklist if deepslateGenerationMode=1 so I could make it so deepslate only tries to generate in blobs in dimensions I specify.

**Comment by Roadhog360** [comment]
That option is specifically for the layer. If I did it for the blobs then I would have to do it for every ore type and blob generation type and I don't want to do that
